integration_tests/suite/test_config.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2018-2022 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+
+from hamcrest import (
+    assert_that,
+    has_key,
+    calling,
+    has_properties,
+)
+
+
+from wazo_dird_client.exceptions import DirdError
+from wazo_test_helpers.hamcrest.raises import raises
+
+
+from .helpers.wait_strategy import EverythingOkWaitStrategy
+
+
+from .helpers.base import (
+    BaseDirdIntegrationTest,
+    VALID_TOKEN,
+    VALID_SUB_TOKEN,
+    MASTER_TENANT,
+    SUB_TENANT,
+)
+
+
+class TestConfig(BaseDirdIntegrationTest):
```

Comment:
Test are missing when wazo-auth is down
You want to be sure that 503 is triggered